lib/util/Floats.v3

```diff
@@ -0,0 +1,665 @@
+// Copyright 2024 Virgil authors. All rights reserved.
+// See LICENSE for details of Apache 2.0 license.
+
+// ====================== Start of Ryu license ========================
```

Comment:
Since both Virgil and Ulf used the Apache 2.0 license, you can combine these notices, e.g.

```
// Copyright 2025 Virgil authors.
// Copyright 2018 Ulf Adams (see below).

...

// Based on code by Ulf Adams (further reference)
```